Update opentelemetry-tutorial-java.mdx (#14812)
newrelic/docs-website: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx

      * `OTEL_EXPORTER_OTLP_HEADERS=api-key=INSERT_YOUR_NEW_RELIC_LICENSE_KEY`
      * `OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317`

> 3. Set these environment variables to name the service, set the service instance ID (which will help light up certain platform features), and enable logs (logging is off by default):

Reviewer: ```suggestion
3. Set these environment variables to name the service, then set the service instance ID. The service instance ID activates certain platform features. Once variables and IDs are set, enable logs since logging is off by default:
```
